bundles/AdminBundle/Resources/public/css/admin.css

```diff
@@ -1184,7 +1184,7 @@ span.warning {
 .pimcore_editable_wysiwyg {
     position:relative;
     padding: 10px;
-    min-height: 80px;
+    min-height: 100%;
```

Comment:
I would suggest that we leave the minimum height at 80px. Otherwise, if no height is specified for the editor, as in the demo on the Car Class, the editor is quite small. 

```suggestion
    min-height: 80px;
    height: 100%;
```

Reply:
That works as well, but I get a slight overflow which causes a scrollbar with that, on focus. Not sure if that id desirable.

Reply:
I think this could do it https://caniuse.com/css-math-functions